apps/site-landing/src/components/Renderer/ElementRender.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { RenderableType } from "components/Prismic/types";
+import { Header } from "semantic-ui-react";
+
+export const ElementRender = ({type, text}: RenderableType) => {
+
+  switch(type)
+  {
+    case "heading1":
+      return <Header as={"h1"}>{text}</Header>;
+    case "heading2":
+      return <Header as={"h2"}>{text}</Header>;
+    case "heading3":
+      return <Header as={"h3"}>{text}</Header>;
+    case "heading4":
+      return <Header as={"h4"}>{text}</Header>;
+    case "list-item":
```

Comment:
Just curious - but why not `<li>`?

Reply:
It was that way before and I assume it would be difficult to add the ul before with the li. I hope to be able to remove those I am not sure they are actually used that much